### PR TITLE
[IMP] mail: fine-tune activity view cells

### PR DESCRIPTION
**Pull request by chgo-odoo**
| Header | Header |
|--------|--------|
| image | image | 

This PR finetunes the cells of the activity view after the webclient redesign. It slightly reduces the strength of the gradient applied on the cells and adds a bit of padding.

task-6577089

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by chgo-odoo** [comment]
Hello @alexkuhn 👋 

We've this minor fine-tuning for the activity view, if you could have a look at it, not very consistent but nice to have 😅

**Review by alexkuhn — APPROVED** [review]
@robodoo delegate+ r+
